scripts/announce.py

```diff
@@ -67,6 +67,17 @@ def get_authors(revision_range):
     cur.discard('Homu')
     pre.discard('Homu')
 
+    # Update doc/source/names_wordlist.txt with the names of every author
+    names = []
+    [names.extend(re.sub('\W+', ' ', x).split()) for x in cur.union(pre)]
+
+    path = os.path.sep.join(os.path.abspath(__file__).split(os.path.sep)[:-2])
```

Comment:
Can use `os.path.dirname` to go up a couple directories or alternately `os.path.join` using `../..` (latter is more succinct though not completely sure of portability)